kivy/uix/image.py

```diff
@@ -400,3 +401,74 @@ def reload(self):
             Loader.remove_from_cache(source)
 
         super(AsyncImage, self).reload()
+
+
+class Base64Image(Image):
+
+    __events__ = ('on_error', 'on_load')
+
+    def __init__(self, **kwargs):
+        self._coreimage = None
+        super(Base64Image, self).__init__(**kwargs)
+        self.fbind('source', self._load_source)
+        if self.source:
+            self._load_source()
+        self.on_anim_delay(self, kwargs.get('anim_delay', .25))
+
+    def _load_source(self, *args):
+        source = self.source
+        if not source:
+            if self._coreimage is not None:
+                self._coreimage.unbind(on_texture=self._on_tex_change)
+                self._coreimage.unbind(on_load=self._on_source_load)
+            self.texture = None
+            self._coreimage = None
+        else:
+            try:
+                image64 = source.split(';base64,')[1]
+                type64 = source.split('data:image/')[1] \
+                                .split(';base64,')[0]
+            except:
```

Comment:
that exception catching is a bit too broad to me, are you looking for `IndexError` instead?

Reply:
Now when you say it!

Reply:
addressed in daab224